### ISSUE TITLE
Process finished with exit code 137 (interrupted by signal 9: SIGKILL)

### ISSUE THREAD

**Issue by khawar512** [serengil/deepface#152]
I am doing experiment on large scale datasets with extensive memory and GPU. Actually, when i started an experiments on Ensemble-Face-Recognition my process has been killed. Is it possible to consume my GPU for these experiments.
```
(236167, 3)
Process finished with exit code 137 (interrupted by signal 9: SIGKILL)
```

**Comment by khawar512** [comment]
I want to used GPU because process has been killed again and again

**Comment by serengil** [comment]
Then, you can set is as shown below.

```
import os
os.environ["CUDA_VISIBLE_DEVICES"]="0" 
```

**Comment by khawar512** [comment]
In my opinion pandas and numpy do not use GPU. Am i right? Because i tried you above but it did not work.

**Comment by serengil** [comment]
Yes, numpy and pandas don't use gpu. GPU is available for keras models in deepface.

**Comment by khawar512** [comment]
What about ensemble-Face-Recognition.py?

**Comment by serengil** [comment]
It will build lightgbm model as well and it can run gpu as well. Could you share the detailed error message?

**Comment by khawar512** [comment]
Actually, i just got this error. I takes 3 minutes and then this error occurred  .


**(236167, 3)
Process finished with exit code 137 (interrupted by signal 9: SIGKILL)**

**Comment by khawar512** [comment]
Meanwhile, I just used one model and one distance metric

**Comment by serengil** [comment]
You got still error when you disable GPU? That's all?

**Comment by khawar512** [comment]
Yes in both conditions, i got an error

**Comment by serengil** [comment]
Could you share the code you are running?

**Comment by serengil** [comment]
It seems that your python program got this error. Your dictionary is too large.

**Comment by pankhurijain20** [comment]
I am facing the same error for Mask-RCNN model. It is unable to run even single epoch
